.eslintrc.js

```diff
@@ -21,6 +21,14 @@ module.exports = {
       files: ['*.test.ts', '*.test.js'],
       extends: ['@metamask/eslint-config-jest'],
     },
+
+    {
+      files: ['jest.config.ts'],
+      rules: {
+        // TODO: Migrate to our shared config
+        'import/no-anonymous-default-export': 'off',
```

Comment:
:eyes: 

I was curious what this rule was for so I just looked up the original PR where it was added. Apparently the advantage is that it "improves the greppability of the codebase". That is, when searching for an identifier, sometimes it can be hard to find what you're looking for if that identifier is only in the filename and not represented in the file itself.

Reply:
I have definitely had that problem myself, so in general this rule seems beneficial to me. But it does seem out-of-place when applied to a top-level config file.